### ISSUE TITLE
Model docs build failing on linspace error

### ISSUE THREAD

**Issue by smk78** [SasView/sasmodels#383]
While building the model docs I got this error:

```
"python" genmodel.py ../sasmodels/models/onion.py model/onion.rst
Traceback (most recent call last):
  File "D:\Anaconda3\envs\qt5\lib\site-packages\numpy\core\function_base.py", line 117, in linspace
    num = operator.index(num)
TypeError: 'numpy.float64' object cannot be interpreted as an integer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "genmodel.py", line 286, in <module>
    main()
  File "genmodel.py", line 283, in main
    process_model(infile, outfile)
  File "genmodel.py", line 279, in process_model
    make_figure(model_info, opts)
  File "genmodel.py", line 143, in make_figure
    plot_profile_inset(model_info, ax1d)
  File "genmodel.py", line 87, in plot_profile_inset
    x, y, labels = call_profile(model_info)
  File "D:\SasViewCodeDev\sasmodels\sasmodels\direct_model.py", line 99, in call_profile
    x, y = model_info.profile(**args)
  File "D:\SasViewCodeDev\sasmodels\sasmodels\models\onion.py", line 378, in profile
    for z_shell in np.linspace(0, thickness[k], num_steps+1):
  File "<__array_function__ internals>", line 6, in linspace
  File "D:\Anaconda3\envs\qt5\lib\site-packages\numpy\core\function_base.py", line 121, in linspace
    .format(type(num)))
TypeError: object of type <class 'numpy.float64'> cannot be safely interpreted as an integer.
make: *** [model/onion.rst] Error 1
```

Also see https://github.com/SasView/sasview/issues/1453 and https://github.com/SasView/sasview/pull/1479

**Comment by pkienzle** [comment]
Fixed in PR #382

**Comment by smk78** [comment]
Built branch locally. All models compiled fine. Closing this issue.
